GOVERNANCE.md

```diff
@@ -2,133 +2,140 @@
+  an issue/question to the right person.
+* Make sure that ongoing PRs are moving forward at the right pace or closing them
+* Monitor Rook email aliases
+* Monitor Rook Slack (delayed response is perfectly acceptable), particularly for the area of your
+  storage provider
+* Regularly attend the recurring [community meetings](README.md#community-meeting)
+* Periodically attend the recurring steering committee meetings to provide input
+* In general continue to be willing to spend at least 25% of ones time working on Rook (~1.25
+  business days per week)
+
+The current list of maintainers is published and updated in [OWNERS.md](OWNERS.md#maintainers).
+
+### Reviewer Role
+
+Reviewers have similar responsibilities as maintainers, with the differences listed in the [Change Approval](CONTRIBUTING.md#change-approval)
+roles of the developer guide. Rules for adding and removing reviewers will follow the same guidelines as
+adding and removing maintainers as described below.
+
+### Becoming a maintainer
 
 To become a maintainer you need to demonstrate the following:
 
-* A strong commitment to the project
-  * Participate in design and technical discussions
-  * Contribute non-trivial pull requests
-  * Perform code reviews on other's pull requests
-  * Answer user questions and troubleshoot issues in the field
-* Ability to write good solid code
-* Ability to collaborate with the team
-* Understanding of how the team works (policies, processes for testing and code review, etc.)
-* Understanding of the project's code base and coding style
-
-### Your organization is not yet a maintainer
-
-* Express interest to the [senior maintainers](OWNERS.md#senior-maintainers) directly that your
-  organization is interested in becoming a maintainer. Becoming a maintainer generally means that
-  you are going to be spending substantial time (>25%) on Rook for the foreseeable future. You
-  should have domain expertise and be extremely proficient with Kubernetes and Golang.  Ultimately
-  your goal is to become a senior maintainer that will represent your organization.
-* You should likely have already been commenting on pull requests and issues with the intent of solving
-  user problems and improving the overall quality of the project.
-* We will expect you to start contributing increasingly complicated PRs, under the guidance
-  of the existing senior maintainers.
+* Consistently be seen as a leader in the Rook community by fulfilling the Maintainer responsibilities
+  listed above to some degree.
+* Domain expertise for at least one of the Rook storage providers
+* Be extremely proficient with Kubernetes and Golang
+* Consistently demonstrate:
+  * Ability to write good solid code
+  * Ability to collaborate with the team
+  * Understanding of how the team works (policies, processes for testing and code review, etc.)
+  * Understanding of the project's code base and coding style
+.
```

Comment:
this period here is causing the markdown to be rendered incorrectly by including the next section ("Beyond your contributions...") within the list above